packages/components/e2e/component-objects/src/test/java/org/talend/config/WebDriverTestFactory.java

```diff
@@ -0,0 +1,15 @@
+package org.talend.config;
+
+public class WebDriverTestFactory {
+
+    public WebDriverTest getWebDriverTestConfiguration() {
+        boolean isRemote = Boolean.parseBoolean(System.getProperty("remote", "false"));
+        if (isRemote) {
+            System.out.println("Remote WebDriver test configuration enabled");
```

Comment:
Why didn't you used the `Logger` used in other classes ?

Reply:
👌